system-addon/content-src/components/Base/Base.jsx

```diff
@@ -137,7 +140,9 @@ export class BaseContent extends React.PureComponent {
             </div>
             <ConfirmDialog />
           </main>
-        </div>);
+        </div>
+        {showOverlay}
```

Comment:
Just do:
` {props.isFirstRun && <StartupOverlay />}`
and remove the showOverlay stuff